hack/install-e2e-dependencies.sh

```diff
@@ -25,11 +25,14 @@ docker run -v /usr/local/bin:/hostbin quay.io/jetstack/ubuntu-nsenter cp /nsente
 
 # Create a cluster. We do this as root as we are using the 'docker' driver.
 # We enable RBAC on the cluster too, to test the RBAC in Navigators chart
-sudo -E CHANGE_MINIKUBE_NONE_USER=true minikube start \
+if ! sudo -E CHANGE_MINIKUBE_NONE_USER=true minikube start \
      -v 100 \
      --vm-driver=none \
      --kubernetes-version="$KUBERNETES_VERSION" \
-     --extra-config=apiserver.Authorization.Mode=RBAC
+     --extra-config=apiserver.Authorization.Mode=RBAC \
+     --bootstrapper=kubeadm; then
+     sudo journalctl -xu kubelet.service
```

Comment:
This is failing on Travis because the VM doesn't have `journalctl`.
Use `minikube logs` instead? 
Or does `kubeadm` put the logs somewhere where `minikube logs` can't find them?

Reply:
This is still WIP - we can't use the kubeadm bootstrapper with vm-driver=none as kubeadm requires systemctl, but travis don't support anything newer than ubuntu 14.04.

I'm working on getting rid of Travis in favour of minikube-in-go on our own test-infra at the moment.